plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -2407,6 +2407,55 @@ public void testDynamicFilteringCaseInsensitiveDomainCompaction()
         }
     }
 
+    @Test
+    public void testUpdateProcedure()
+    {
+        String tableName = "test_update" + randomNameSuffix();
+        String schemaTableName = getSession().getSchema().orElseThrow() + "." + tableName;
+
+        assertUpdate("CREATE TABLE " + schemaTableName + "(a int)");
+        try {
+            assertUpdate("CALL system.update('INSERT INTO " + schemaTableName + " VALUES (1)')");
+            assertQuery("SELECT * FROM " + schemaTableName, "VALUES 1");
+
+            assertUpdate("CALL system.update('UPDATE " + schemaTableName + " SET a = 2')");
```

Comment:
Wouldn't the customer want to get the amount of rows affected by the statement?

Reply:
The engine doesn't support returning affected rows in procedures. See `CallTask` and `DataDefinitionExecution` for more details.